public/js/Constants.js

```diff
@@ -14,8 +14,18 @@ var Constants: {
 
   resourceSize: number,
   numResources: number,
-
   maxHistoryBuffer: number,
+
+  animals: Array<string>,
+  adjectives: Array<string>,
```

Comment:
name_prefix_adjectives might be more descriptive to reduce confusion in the future